WordPress/gutenberg
# [RNMobile][Embed block] Add device's locale to preview content

`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3788





## Description


Use the device's locale in the HTML generated for the embed preview.

## How has this been tested?




**NOTE:** For testing these changes, the embed inline preview has to be available.

1. Change the device's language to a non-English one (in my case I used Spanish)
2. Add an Embed block
3. Set a valid embed URL
4. Observe if the preview's content is localized to the selected language

**NOTE:** Each embed provider supports different languages so it's recommended to verify how is rendered in the web version.

## Screenshots 

**Using Spanish as device's language:**

Android|iOS
--|--
<img src=https://user-images.githubusercontent.com/14905380/132042703-3c95c772-ed6e-4e6b-a6cf-e836e9923c61.jpg>|<img src=https://user-images.githubusercontent.com/14905380/132042705-a600ed14-0b06-4b7e-b51f-77c48c1997dc.png>

## Types of changes




New feature

## Checklist:
